mobile/src/libs/enums/ui/app-color.enum.ts

```diff
@@ -7,6 +7,9 @@ const AppColor = {
   GRAY_200: '#CCDDF3',
   GRAY_300: '#ACB4D0',
   GRAY_400: '#302E36',
+  GRAY_500: '#1D1C21FF',
+  TRANSPARENT_GRAY_100: '#FFFFFF7F',
+  TRANSPARENT_GRAY_200: '#312F377F',
 } as const;
 
```

Comment:
please change whole file to this 
const AppColor = {
  WHITE: 'white',
  BLACK: 'black',
  BLUE_100: '#4E80FF',
  BLUE_200: '#507CEB',
  BLUE_300: '#113182',
  PINK_100: '#E1A7D5',
  RED_100: '#D51A52',
  GRAY_100: '#EBF3FF',
  GRAY_200: '#CCDDF3',
  GRAY_300: '#ACB4D0',
  GRAY_400: '#302E36',
  GRAY_500: '#1D1C21FF',
  GRAY_100_ALPHA_50: '#FFFFFF7F',
  GRAY_200_ALPHA_50: '#312F377F',
  BLUE_100_ALPHA_70: '#A6D4FFB2',
} as const;

export { AppColor };
and please change color using in styles to these new names ( pay attention to all white colors)

Reply:
Isn't it better to use Hex Codes everywhere for consistency sake (#ffffff instead of white)?

Reply:
Yeah, we should only use hex